### PR TITLE
fix alternate templates for 0T cosmics (CRUZET'15).

### PR DESCRIPTION
**Pull request by tlampen**

### CODE REVIEW

File: Alignment/OfflineValidation/python/TkAlAllInOneTool/alternateValidationTemplates.py [hunk 1]
```diff
@@ -441,14 +442,11 @@
    TrajectoryInEvent = True,
    TTRHBuilder = "WithAngleAndTemplate",
    NavigationSchool = "",
-   constraint = 'momentum', ### SPECIFIC FOR CRUZET
-   srcConstr='AliMomConstraint1' ### SPECIFIC FOR CRUZET$works only with tag V02-10-02 TrackingTools/PatternTools / or CMSSW >=31X
```
Comment (mmusich) [inline Alignment/OfflineValidation/python/TkAlAllInOneTool/alternateValidationTemplates.py]: @tlampen why is the momentum constraint removed from the track refit?

**Comment by mmusich** [comment]
hold
This needs to be fixed from the POG on the `KFTrajectoryFitter` side

**Comment by mmusich** [comment]
in the light of PR #8342 from Vincenzo, can you please put back the momentum constraints instructions to the TrackRefiter.

**Comment by tlampen** [comment]
PR #8342 from Vincenzo fixes this, so this PR is not needed.

**Comment by mmusich** [comment]
@tlampen can you clarifies if `NavigationSchool = ""` is also not needed? I think it was decided this is the baseline configuration.

**Pull request closed without merging**